### PR TITLE
[Wallet] Add shake to Contact screen

### LINKED ISSUE #5689: Add shake to send logs functionality to the wallet
### Expected Behavior

When shaking the device, the user is directed to the Contact screen so that they can share logs.

### Current Behavior

Shaking does nothing.

### PR DESCRIPTION
**Pull request by gnardini**
### Description

When shaking the device, the user is now directed to the Contact screen so that they can share logs.

### Tested

On physical Android and iOS devices by shaking them.

### Related issues

- Fixes #5689

### CODE REVIEW

**Comment by gnardini** [comment]
Open question: 
The current implementation redirects users to the contact screen always when shaking. Is that desirable or would it better for this to happen only during the onboarding flow?

**Comment by jeanregisser** [comment]
> Open question:
> The current implementation redirects users to the contact screen always when shaking. Is that desirable or would it better for this to happen only during the onboarding flow?

I think that's fine!

File: packages/mobile/src/navigator/NavigatorWrapper.tsx [hunk 1]
```diff
@@ -97,6 +99,15 @@ export const NavigatorWrapper = () => {
...
   }, [isReady])
 
+  React.useEffect(() => {
+    RNShake.addEventListener('ShakeEvent', () => {
+      navigate(Screens.SupportContact)
+    })
+    return () => {
+      RNShake.removeEventListener('ShakeEvent')
+    }
+  })
```
Comment (jeanregisser) [inline packages/mobile/src/navigator/NavigatorWrapper.tsx]:
```suggestion
  }, [])
```

Otherwise it's gonna subscribe/unscribe on each render.

**Review by jeanregisser — CHANGES_REQUESTED** [review]
Awesome 💯 
Almost good to merge.

File: packages/mobile/src/navigator/NavigatorWrapper.tsx (hunk 1 above)
Comment (gnardini) [reply packages/mobile/src/navigator/NavigatorWrapper.tsx]: Whoops, that's really bad from me :(

Comment (jeanregisser) [reply packages/mobile/src/navigator/NavigatorWrapper.tsx]: No worry, it's easy to miss, I think there's a lint rule we could add to make sure it's present.

**Review by jeanregisser — APPROVED** [review]
Ship it! :shipit:

**Comment by Lss-Ankit** [comment]
Hi @gnardini , @jeanregisser  Verified task using Test flight build v1.4.0 (28) and Android play store (internal build) v1.4.0 (1004294317) and found the following: 

- User is redirect to Contact with text field and attach log screen. 
- Also, observed that user is not redirected to Contact screen when user shaking the phone if user is on Enter PIN screen. 
Issue is repro only when user is on Enter PIN screen from Account key screen. 
Issue does not repro if user is on Enter PIN screen while sending payment. 

[Attachment: ](https://drive.google.com/file/d/1A7QEVCBuBRpNnS0uawSKyfyP91awffNQ/view?usp=sharing)
Can you please confirm should we file a bug for the same?

**Comment by gnardini** [comment]
I think it's not necessary to file a bug for this since the main goal was for it to be used while on the onboarding flow.

**Comment by Lss-Ankit** [comment]
ohk thanks for the confirmation!!

**Pull request merged**